src/js/tracker.js

```diff
@@ -1780,67 +1745,6 @@ export function Tracker(functionName, namespace, version, mutSnowplowState, argm
     );
   }
 
-  /**
```

Comment:
Similar thing here - any context on what this code is for in the first place?

Reply:
I had to do a bit of Googling on this one to see what it was all about. 

Once upon a time, there was a visility state called `prerendered` that a page could be in, and it wouldn't be visible to a user. We used to allow prerendered pages to be ignored or included which is what this code used to do based on `configCountPreRendered` and we'd only trigger the page view (or whatever event) once the page transitioned into another visibility state (see the event listenter in this section).

However, this has been deprecated for a while and has poor cross browser support. From [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Document/visibilityState):
> prerender
> The page content is being prerendered and is not visible to the user (considered hidden for purposes of document.hidden). The document may start in this state, but will never transition to it from another value. Note: This was removed from the standard. Check compatibility table for details.

However, you pointing this out has made me realise I missed the config variable!